packages/twenty-server/src/engine/core-modules/workspace/workspace.resolver.ts

```diff
@@ -124,6 +130,19 @@ export class WorkspaceResolver {
     return await this.userWorkspaceService.getUserCount(workspace.id);
   }
 
+  @ResolveField(() => String)
+  async logo(@Parent() workspace: Workspace): Promise<string> {
+    if (workspace.logo) {
+      const workspaceLogoToken = await this.fileService.encodeFileToken({
+        workspace_id: workspace.id,
+      });
+
+      return `${workspace.logo}?token=${workspaceLogoToken}`;
+    }
+
+    return workspace.logo ?? '';
```

Comment:
I think it will always return '' here right? @charlesBochet 
Or you can
```typescript
async logo(@Parent() workspace: Workspace): Promise<string> {
  if (!workspace.logo) {
    return '';
  }

  const workspaceLogoToken = await this.fileService.encodeFileToken({
    workspace_id: workspace.id,
  });

  return `${workspace.logo}?token=${workspaceLogoToken}`;
}
```